File: Core/CMS.Domain/Entities/ContractsCount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CMS.Domain.Entities
{
    public class ContractsCount
    {
        public int AllContractsCount { get; set; }
        public int PendingApprovalContractsCount { get; set; }
        public int PendingTerminationContractsCount { get; set; }
        public int ExpiredContractsCount { get; set; }
        public int ActiveContractsCount { get; set; }
        public int TerminatedContractsCount { get; set; }
    }
}

File: Frontend/CMS/src/app/models/contracts-count.ts
export class ContractsCount {
    allContractsCount: number;
    pendingApprovalContractsCount: number;
    pendingTerminationContractsCount: number;
    expiredContractsCount: number;
    activeContractsCount: number;
    terminatedContractsCount: number;
    constructor(
        allContractsCount: number,
        pendingApprovalContractsCount: number,
        pendingTerminationContractsCount: number,
        expiredContractsCount: number,
        activeContractsCount: number,
        terminatedContractsCount: number,
    ) {
        this.allContractsCount = allContractsCount;
        this.pendingApprovalContractsCount = pendingApprovalContractsCount;
        this.pendingTerminationContractsCount = pendingTerminationContractsCount;
        this.expiredContractsCount = expiredContractsCount;
        this.activeContractsCount = activeContractsCount;
        this.terminatedContractsCount = terminatedContractsCount;
    }
}
